util/cron/create_release_tarball.bash

```diff
@@ -35,10 +35,10 @@ if [ "${CHPL_VERSION}" = "detect" ] ; then
         exit 1
     fi
 
-    version_num_text=$(cat $CHPL_HOME/compiler/main/version_num.h)
-    major=$(echo "${version_num_text}" | grep MAJOR_VERSION | cut -f3 -d" " | sed "s/\"//g")
-    minor=$(echo "${version_num_text}" | grep MINOR_VERSION | cut -f3 -d" " | sed "s/\"//g")
-    patch=$(echo "${version_num_text}" | grep UPDATE_VERSION | cut -f3 -d" " | sed "s/\"//g")
+    version_num_text=$(cat $CHPL_HOME/CMakeLists.txt)
+    major=$(echo "${version_num_text}" | grep 'set(CHPL_MAJOR_VERSION' | cut -f2 -d" " | sed "s/\)//g")
```

Comment:
The logic for determining these is complicated enough and appears in enough bash scripts that I think it warrants extracting out to a separate common file where it can be retrieved